web-ui/src/pages/SelfReviewsPage.jsx

```diff
@@ -1,57 +1,61 @@
-import React, {useContext, useCallback, useEffect, useState} from "react";
-import { styled } from '@mui/material/styles';
-import Typography from "@mui/material/Typography";
-import { useLocation, useHistory } from 'react-router-dom';
+import React, {useContext, useCallback, useEffect, useState} from 'react';
+import {styled} from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import {useLocation, useHistory} from 'react-router-dom';
 import queryString from 'query-string';
-import {AppContext} from "../context/AppContext";
-import {selectCurrentUser} from "../context/selectors";
-import ReviewPeriods from "../components/reviews/periods/ReviewPeriods";
-import SelfReview from "../components/reviews/SelfReview";
+import {AppContext} from '../context/AppContext';
+import {selectCurrentUser} from '../context/selectors';
+import ReviewPeriods from '../components/reviews/periods/ReviewPeriods';
+import SelfReview from '../components/reviews/SelfReview';
 
 const PREFIX = 'SelfReviewPage';
 const classes = {
   root: `${PREFIX}-root`,
   headerContainer: `${PREFIX}-headerContainer`,
   requestHeader: `${PREFIX}-requestHeader`,
-  stepContainer: `${PREFIX}-stepContainer`,
+  stepContainer: `${PREFIX}-stepContainer`
 };
 
 const Root = styled('div')(({theme}) => ({
   [`&.${classes.root}`]: {
-    backgroundColor: "transparent",
-    padding: "4rem 2rem 0 2rem",
-    height: "100%",
-    'max-width': "100%",
```

Comment:
I changed all CSS properties that do not use camelCase names to use them in order to remove warnings.